src/applications/personalization/profile/components/alerts/ContactInfoNeeded.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { selectVAPContactInfoField } from '@@vap-svc/selectors';
+import {
+  VaAlert,
+  VaLinkAction,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { Toggler } from 'platform/utilities/feature-toggles';
+
+const ContactInfoNeeded = () => {
+  const email = useSelector(state => selectVAPContactInfoField(state, 'email'));
+  const mailingAddress = useSelector(state =>
+    selectVAPContactInfoField(state, 'mailingAddress'),
+  );
+  const mobilePhone = useSelector(state =>
+    selectVAPContactInfoField(state, 'mobilePhone'),
+  );
```

Comment:
Non-blocking

I usually try to stay away from 'magic strings' like email, mailingAddress, and mobilePhone. There is a constant called FIELD_NAMES that you could use for getting these names.

`import { FIELD_NAMES } from 'platform/user/profile/vap-svc/constants';`